frontend/src/pages/DashboardsListPage/__tests__/DashboardListPage.test.tsx

```diff
@@ -204,4 +212,25 @@ describe('dashboard list page', () => {
 			),
 		);
 	});
+
+	it('ensure that the export JSON popover action works correctly', async () => {
+		const { getByText, getAllByTestId } = render(
+			<MemoryRouter initialEntries={['/dashbords']}>
+				<DashboardProvider>
+					<DashboardsList />
+				</DashboardProvider>
+			</MemoryRouter>,
+		);
+
+		await waitFor(() => {
+			const popovers = getAllByTestId('dashboard-action-icon');
+			expect(popovers).toHaveLength(dashboardSuccessResponse.data.length);
+			fireEvent.click([...popovers[0].children][0]);
+		});
+
+		const exportJsonBtn = getByText('Export JSON');
+		expect(exportJsonBtn).toBeInTheDocument();
+		fireEvent.click(exportJsonBtn);
+		expect(dashboardUtils.sanitizeDashboardData).toHaveBeenCalled();
```

Comment:
we can check to `haveBeenCalledWith(expectedResponse)` , will help to better check the final state